### DISCUSSION TITLE
Generating template from a local folder without using git

### DISCUSSION THREAD

**Discussion by Savlik** [copier-org/copier#550] [Q&A] [answered]
Hi.
I would like to generate a template from a local folder, but there is git inited there and it automatically assumes I want to use git and uses current HEAD ref instead of local files present. Is there a way to force Copier into using the local files and ignore git? I can imagine a workaround copying content into temp folder and running it from there, but it is a bit hacky.

Context: I have tests on my template to check if it can be generated. I use pytest with copier library. But now it ignores all unstaged changes, which is a space for mistakes to happen.
```python
from pathlib import Path
from copier import run_copy

def test_generate(tmp_path: Path) -> None:
    run_copy(
        str(Path(__file__).resolve().parent.parent),
        str(tmp_path.resolve()),
        data={...},
        defaults=True,
        quiet=True,
    )
```

**Comment by pawamoy** [comment, marked as answer]
I think https://github.com/copier-org/copier/pull/520 is what you want. Not yet released. You could install copier from the master branch.

**Reply by Savlik** [reply]
Great, that's exactly it, thank you. I will wait for the next v6 pre-release then.
